README.md

```diff
@@ -111,6 +111,7 @@ before.
 10. [`#[derive(AddAssign)]`](https://jeltef.github.io/derive_more/derive_more/add_assign.html)
 11. [`#[derive(MulAssign)]`](https://jeltef.github.io/derive_more/derive_more/mul_assign.html)
 12. [`#[derive(Constructor)]`](https://jeltef.github.io/derive_more/derive_more/constructor.html)
+13. [`#[derive(TryInto)]`](https://jeltef.github.io/derive_more/derive_more/try_into.html)
```

Comment:
Can you change this in the `src/lib.rs` file instead? I like to update the README only when I create a release.

Reply:
Also can you add a link to the "conversion traits" section above?